config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BuilderConstants.java

```diff
@@ -25,7 +28,15 @@ private BuilderConstants(){
     public static final Predicate<Entity> FILTER_ENV_ENTITIES = new Predicate<Entity>() {
         @Override
         public boolean test(Entity entity) {
-            return !NON_ENV_ENTITY_TYPES.contains(entity.getType());
+            return !NON_ENV_ENTITY_TYPES.contains(entity.getType()) &&
+                    !(EntityTypes.LISTEN_PORT_TYPE.equals(entity.getType()) && (DEFAULT_HTTP_8080.equals(entity.getName()) || DEFAULT_HTTPS_8443.equals(entity.getName())));
```

Comment:
Can you move these new tests to another Filter instead of overloading one? And the called can do 
`.filter(FILTER_ENV_ENTITIES.and(<new-predicate-name>))`

Reply:
done